Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -619,7 +776,47 @@ public void UpdateAngleCockState(bool AngleCockOpen, ref float AngleCockOpenAmou
         public override void Update(float elapsedClockSeconds)
         {
             var valveType = (Car as MSTSWagon).BrakeValve;
-            float threshold = valveType == MSTSWagon.BrakeValveType.Distributor ? Math.Max((ControlResPressurePSI - BrakeLine1PressurePSI) * AuxCylVolumeRatio, 0) : 0;
+
+            // Two stage braking: higher brake force is allowed at higher speeds
```

Comment:
For European high speed trains this is the opposite. When speed exceeds 160-200km/h the brake force is reduced, not increased. Could you rewrite the implementation not assuming that LowSpeedPressure < HighSpeedPressure, but allow both regimes?

It doesn't make sense to me that higher brake forces are allowed at higher speeds since the adhesion will be lower (is this to achieve a constant brake force taking into account the reduction in brake shoe friction?)

Reply:
The systems I was looking at were indeed concerned about low brake shoe friction at speed. I'll look into allowing the relationship to be reversed, but using the two stage relay ratio feature should already allow for lower or higher braking force depending on how you set it up. And I think the relay ratio approach is a bit more accurate than the flat pressure limit for most systems.

Reply:
Ah, if using a higher relay ratio for low speeds work, then it's only necessary to update the feature description (i.e. manual and code comments)

Reply:
I did some testing in the current unstable build and this token seems to work just fine if you use ORTSMaxServiceCylinderPressure to set the higher speed pressure lower than ORTSTwoStageLowPressure for the low speed pressure. I'll make sure the documentation reflects that this works fine whether set lower or higher than the service max pressure.